cumulus/primitives/utility/src/lib.rs

```diff
@@ -51,6 +52,17 @@ impl<T: Get<MultiAssets>> PriceForParentDelivery for ConstantPrice<T> {
 	}
 }
 
+impl<A: Get<AssetId>, B: Get<u128>, M: Get<u128>, F: FeeTracker> PriceForParentDelivery
+	for ExponentialPrice<A, B, M, F>
+{
+	fn price_for_parent_delivery(message: &Xcm<()>) -> MultiAssets {
+		let message_fee = (message.encoded_size() as u128).saturating_mul(M::get());
+		let fee_sum = B::get().saturating_add(message_fee);
+		let amount = F::get_fee_factor(()).saturating_mul_int(fee_sum);
+		(A::get(), amount).into()
+	}
+}
```

Comment:
Just from looking at this, it's not really clear what `B` and `M` refer to

Reply:
Removed that code since it was a duplicate of the other implementation of `ExponentialPrice`, which did have docs